xclim/indicators/atmos/_precip.py

```diff
@@ -308,3 +310,23 @@ class HrPrecip(Hourly):
         indices.liquid_precip_ratio, suggested={"tas": _empty}, prsn=None
     ),
 )
+
+
+rolling_drydays_events = Precip(
+    identifier="rolling_drydays_events",
+    description="The number of periods of minimum {window} days during which the accumulated precipitation is under"
+    "{thresh}, for each resampling period.",
```

Comment:
```suggestion
    description="The {freq} number of periods of minimum {window} days during which the accumulated precipitation is under {thresh}.",
```
Formatting trick: common `freq` values are translated to an adjective when inserted in the attributes. Allows for less words and more precision!

Reply:
`description="The {freq} number of periods of at least {window} days during which accumulated {window} day precipitation is under {thresh}.",`

Would this suggested change clarify the nuance that say a 9 or 10 day period is a single event even if total precip over all days is greater than thresh?   Or does it make it more difficult?